### ISSUE TITLE
No neon icons?

### ISSUE THREAD

**Issue by Mattevt** [gusbemacbe/suru-plus#148]
I followed a link here hoping to find icons to complement a neon Ubuntu theme. But all the icons are really vanilla iOS-type rounded squares. Am I missing something?

**Comment by dror-g** [comment]
Yes, the icons are not colorized by default. you need to use [suru-plus-colorize](https://github.com/gusbemacbe/suru-plus-colourise)


It's mentioned in the wiki [here](https://github.com/gusbemacbe/suru-plus/wiki/Changing-the-folder-colour) but perhaps needs to be mentioned in install instructions.

**Comment by gusbemacbe** [comment]
Greetings, @Mattevt and @dror-g!

I officially made a short announcement about the discontinuation and the transfer of ownership.  

To read the reasons for discontinuation and the future of these projects, you will find the explanations linked in [my OpenDesktop topic](https://forum.opendesktop.org/t/i-officially-discontinue-the-suru-icons-theme-i-accept-the-transfer-of-repositories-to-anyone-interested/21239) or in [REASON.md](./[REASON.md](https://github.com/gusbemacbe/suru-plus/blob/master/REASON.md)).
